### PR TITLE
Refactor grpclient, make it generic

### PR DESCRIPTION
**Pull request by jiaoew1991**
Signed-off-by: Enwei Jiao <enwei.jiao@zilliz.com>

/kind improvement

### CODE REVIEW

**Comment by jiaoew1991** [comment]
rerun ut

**Comment by jiaoew1991** [comment]
rerun ut

File: tests/python_client/testcases/test_utility.py [hunk 1]
```diff
@@ -1655,8 +1655,8 @@ def test_load_balance_with_all_sealed_segment_id_not_exist(self):
         dst_node_ids = all_querynodes[1:]
         # add segment ids which are not exist
         sealed_segment_ids = [sealed_segment_id
-                              for sealed_segment_id in range(max(segment_distribution[src_node_id]["sealed"]) + 1,
-                                                             max(segment_distribution[src_node_id]["sealed"]) + 3)]
+                              for sealed_segment_id in range(max(segment_distribution[src_node_id]["sealed"]) + 100,
+                                                             max(segment_distribution[src_node_id]["sealed"]) + 103)]
```
[the remark this answers is not in the document]
Comment (jiaoew1991) [reply tests/python_client/testcases/test_utility.py:1659]: it's unstable, this change will make it stable

**Review by congqixia — APPROVED** [review]
/lgtm

**Pull request merged**